pyface/ui/qt4/action/menu_manager.py

```diff
@@ -42,6 +42,11 @@ class MenuManager(ActionManager, ActionManagerItem):
     # The default action for tool button when shown in a toolbar (Qt only)
     action = Instance(Action)
 
+    # Private interface ---------------------------------------------------#
+
+    #: Keep track of all created menus in order to properly dispose of them
+    _menus = []
```

Comment:
I overlooked this. My apologies. This should be `List()`, otherwise we only get one list that all instances of `MenuManager` share.

Reply:
Opened #548